core/audits/insights/interaction-to-next-paint-insight.js

```diff
@@ -36,14 +34,28 @@ class InteractionToNextPaintInsight extends Audit {
    * @return {Promise<LH.Audit.Product>}
    */
   static async audit(artifacts, context) {
-    // TODO: implement.
     return adaptInsightToAuditProduct(artifacts, context, 'InteractionToNextPaint', (insight) => {
+      const event = insight.longestInteractionEvent;
```

Comment:
Similar to LCP, I think it makes sense to stick the INP element in this insight even though the RPP sidebar doesn't do it.